.php-cs-fixer.dist.php

```diff
@@ -7,14 +7,14 @@
 use PhpCsFixer\Finder;
 
 $finder = Finder::create()
-    ->name('.php_cs.dist') // Fix this file as well
+    ->name('.php-cs-fixer.dist.php') // Fix this file as well
```

Comment:
As this file now ends in `.php` there's no need to specifically add it anymore, this line can thus be removed. Same for the code block in te readme.

Reply:
Fair point! Only added it to be consistent with the current example. I'll remove it